bin/dune_init.ml

```diff
@@ -230,8 +230,6 @@ module Component = struct
           | Exec
           | Lib
 
-        (* TODO(shonfeder) Add custom templates *)
```

Comment:
Removed because I don't think we're planning on this any more. But, in any case, I won't be adding it. So if we still want this TODO, I can just remove my name.

Reply:
I'm all for removing uncertain TODOs